app/domain/authentication/authn_jwt/authentication_parameters.rb

```diff
@@ -0,0 +1,24 @@
+module Authentication
+  module AuthnJwt
+    # Data class to store data regarding jwt token that is needed during the jwt authentication process
+    class AuthenticationParameters
+      attr_writer :jwt_identity
+      attr_reader :decoded_token, :authenticator_name, :service_id, :account, :username, :credentials, :client_ip, :request
+
+      def initialize(authentication_input, decoded_token)
+        @decoded_token = decoded_token
+        @authenticator_name = authentication_input.authenticator_name
+        @service_id = authentication_input.service_id
+        @account = authentication_input.account
+        @username = authentication_input.username
+        @credentials = authentication_input.credentials
+        @client_ip = authentication_input.client_ip
+        @request = authentication_input.request
+      end
+
+      def authenticator_resource_id
+        "#{@account}:variable:conjur/#{@authenticator_name}/#{@service_id}"
```

Comment:
What is this variable?
Why is it here?
I don't see any context to explain its meaning...

Reply:
This is the resouce id in the conjur resource tree where all relevant variables for the jwt authenticatior are placed

Reply:
See the policy here
https://github.com/cyberark/conjur/blob/eac4257c4685ab42ba0744bfb2e73d8d30701c1b/design/authenticators/authn_jwt/authn_jwt_solution_design.md#fetch-identity